src/os/bluestore/bluefs_types.h

```diff
@@ -62,7 +118,7 @@ struct bluefs_fnode_t {
   uint64_t ino;
   uint64_t size;
   utime_t mtime;
-  uint8_t __unused__ = 0; // was prefer_bdev
+  uint8_t type = 0; // was prefer_bdev
```

Comment:
I wouldn't reuse this field in this way - potentially we might still face legacy OSDs that keep non-zero values in this field.